README.md

```diff
@@ -234,6 +244,13 @@ Some general information
         + For type `library` there can be a setting for mapping the virtual paths `src` and `test` to physical paths within the project
 - `shims`: Can be used to define, extend or override UI5 configs of dependencies. Inner structure equals the general structure. It is a key-value map where the key equals the project ID as supplied by the translator.
 
+##### builder (optional)
+
+- `customTasks` (optional, list): in this block you can define additional custom build tasks. Please see ([here](docs/BuildExtensibility.md)) for a detailed explanation and examples of the build extensibility. Each entry in the `customTasks` list consists of the following options:
+  - `name` (mandatory): the name of the custom task
+  - `afterTask` or `beforeTask` (only one, mandatory): the name of the build task after or before which your custom task will be executed.
+  - `configuration` (optional): additional configuration for your custom build task
```

Comment:
> additional configuration for your custom build task

 additional configuration that will be passed to the custom build task

Reply:
Done